packages/x-components/src/utils/__tests__/origin.spec.ts

```diff
@@ -22,12 +16,21 @@ describe(`testing ${createOrigin.name} utility method`, () => {
     ).toBeNull();
   });
 
-  it('returns `feature:location` when the feature and the location are provided', () => {
+  it('returns `feature:location` when the feature is provided', () => {
```

Comment:
I don't think this title is ok, as you are saying `the feature is provided` but the  in this case the location is relevant as it's part of the final origin.